assets/src/components/activities/image_coding/actions.ts

```diff
@@ -41,7 +41,7 @@ export class ICActions {
 
   static addImageURL(value: string) {
     return (draftState: ImageCodingModelSchema) => {
-      draftState.imageURLs.push(value);
+      draftState.resourceURLs.push(value);
```

Comment:
While are you representing the collection of resources as an array, we actually want set semantics here and want to prevent duplicates.

Reply:
OK, these should be addressed now.